### PR TITLE
fix: Progress bar shows correctly in ordersUnderUserFragment

### LINKED ISSUE #930: Progress bar overlapping with tickets
**Describe the bug**
Progress bar overlapping with tickets after returning from ticket detail fragment.

**To Reproduce**
Open a ticket details and go back.

**Screenshots**
<p align="center">
 
</p>

**Additional context**

**Would you like to work on the issue?**

- [x] No

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #930 

Changes: Disabled progress bar when coming back from ticketdetails. 

Screenshots for the change:
ezgif com-video-to-gif

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt [hunk 1]
```diff
@@ -81,7 +81,9 @@ class OrdersUnderUserFragment : Fragment() {
             ordersUnderUserVM.progress
                 .nonNull()
                 .observe(this, Observer {
-                    rootView.progressBar.isVisible = it
+                    if (ordersRecyclerAdapter.attendeesNumber.size==0 && it) {
+                        rootView.progressBar.isVisible = it
+                    } else rootView.progressBar.isVisible = false
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt]: This should only be controlled via ViewModel

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt]: Okay

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt]: done

File: app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserVM.kt [hunk 2]
```diff
@@ -45,7 +45,7 @@ class OrdersUnderUserVM(
             .subscribeOn(Schedulers.io())
             .observeOn(AndroidSchedulers.mainThread())
             .doOnSubscribe {
-                mutableProgress.value = true
+                if (mutableAttendeesNumber.value == null) mutableProgress.value = true
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserVM.kt]:
```suggestion
                mutableProgress.value = mutableAttendeesNumber.value == null
```

**Pull request merged**